geoportal/geoportailv3_geoportal/static-ngeo/js/UserManager.js

```diff
@@ -284,50 +281,71 @@ exports.prototype.setUserInfo = function(
  * @return {string} The username.
  */
 exports.prototype.getUsername = function() {
-  return this.username;
+  const userManagerStore = useUserManagerStore()
+  const { currentUser } = storeToRefs(userManagerStore)
```

Comment:
Could it make sense to instantiate these once in the constructor?

Reply:
done